scripts/sct_image.py

```diff
@@ -18,7 +18,7 @@
 import numpy as np
 
 import spinalcordtoolbox.image as msct_image
-from spinalcordtoolbox.image import Image
+from spinalcordtoolbox.image import Image, concat_data
```

Comment:
Make sense! Super

Reply:
will need to move the other functions too at some point...